### DISCUSSION TITLE
how solve this problem

### DISCUSSION THREAD

**Discussion by tanlee1** [acheong08/EdgeGPT#199] [General]
I user EdgeGPT，but find error "Authentication failed. You have not been accepted into the beta."

how can I solve this problem

**Comment by acheong08** [comment]
Are you in China?

**Comment by tanlee1** [comment]
yes，im chinese
But my service is in Singapore ，
I use the Aliyun server in Singapore
How cani solve this problem

> Are you in China?—

**Reply by acheong08** [reply]
Try updating to https://github.com/acheong08/EdgeGPT/releases/tag/0.1.8

**Reply by acheong08** [reply]
Aliyun, being owned by a Chinese company, is possibly subject to the same restrictions

**Comment by tanlee1** [comment]
Could not find a version that satisfies the requirement EdgeGPT==0.1.8 (from versions: 0.0.2, 0.0.3, 0.0.4, 0.0.5, 0.0.10, 0.0.11, 0.0.20, 0.0.21, 0.0.25, 0.0.30, 0.0.40, 0.0.41, 0.0.41.1, 0.0.42, 0.0.43, 0.0.44, 0.0.45, 0.0.46, 0.0.47, 0.0.48, 0.0.49, 0.0.50, 0.0.51, 0.0.53, 0.0.53.1, 0.0.54, 0.0.55, 0.0.56, 0.0.56.1, 0.0.56.2, 0.0.57, 0.0.58.1, 0.0.58.2, 0.0.59, 0.0.60, 0.1.0, 0.1.1, 0.1.2, 0.1.3)
No matching distribution found for EdgeGPT==0.1.8


I want to upgrade EdgeGPT version but pip only find this highest version is 0.1.3
> From:"Antonio ***@***.***>
> Date:Fri, Mar 24, 2023, 18:32
> Subject:Re: [acheong08/EdgeGPT] how solve this problem (Discussion #199)
> ***@***.***>
> ***@***.***>***@***.***>
> Try updating tohttps://github.com/acheong08/EdgeGPT/releases/tag/0.1.8—
> Reply to this email directly,view it on GitHub<https://github.com/acheong08/EdgeGPT/discussions/199#discussioncomment-5416387>, orunsubscribe<https://github.com/notifications/unsubscribe-auth/A5HIWGE6BD3MUTEO5HVSFH3W5VZ37ANCNFSM6AAAAAAWGH2EME>.
> You are receiving this because you authored the thread.[image: https://github.com/notifications/beacon/A5HIWGDHXT74ZJOJKAPOWTLW5VZ37A5CNFSM6AAAAAAWGH2EMGWGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3TUL5UWJTQAKKS4G.gif]Message ***@***.***>
Please consider the environment before printing this email
Mind Network (MindNet) can be contacted on ***@***.******@***.***>***@***.***>. This email and any attachments are confidential and may be protected by legal privilege. Any use of the information contained herein in any way (including, but not limited to, total or partial disclosure, reproduction, or dissemination) by persons other than the intended recipient(s) is prohibited. If you are not the intended recipient, you must contact us or the sender immediately and then securely delete the email and its attachments. We reserve all rights in respect of any unauthorised use of the email. Anything in this email or its attachments which does not relate to MindNet's official business is neither given nor endorsed by us. We do not necessarily accept legal responsibility for the contents of this email or attachments. Views and opinions expressed are those of the author and do not necessarily represent those of MindNet.

**Reply by acheong08** [reply]
`pip install git+https://github.com/acheong08/EdgeGPT.git` is your cache isn't updating

**Comment by tanlee1** [comment]
So I have to install your git project by myself,
let me try
Thank you very much
I'll let you know if there's a result

> From:"Antonio ***@***.***>
> Date:Fri, Mar 24, 2023, 18:42
> Subject:Re: [acheong08/EdgeGPT] how solve this problem (Discussion #199)
> ***@***.***>
> ***@***.***>***@***.***>
> pip install git+https://github.com/acheong08/EdgeGPT.git
> is your cache isn't updating—
> Reply to this email directly,view it on GitHub<https://github.com/acheong08/EdgeGPT/discussions/199#discussioncomment-5416500>, orunsubscribe<https://github.com/notifications/unsubscribe-auth/A5HIWGDUD5XN7SW3R2VYTQLW5V3CBANCNFSM6AAAAAAWGH2EME>.
> You are receiving this because you authored the thread.[image: https://github.com/notifications/beacon/A5HIWGGXD5O7FUN2QQPHCA3W5V3CBA5CNFSM6AAAAAAWGH2EMGWGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3TUL5UWJTQAKKTDI.gif]Message ***@***.***>
Please consider the environment before printing this email
Mind Network (MindNet) can be contacted on ***@***.******@***.***>***@***.***>. This email and any attachments are confidential and may be protected by legal privilege. Any use of the information contained herein in any way (including, but not limited to, total or partial disclosure, reproduction, or dissemination) by persons other than the intended recipient(s) is prohibited. If you are not the intended recipient, you must contact us or the sender immediately and then securely delete the email and its attachments. We reserve all rights in respect of any unauthorised use of the email. Anything in this email or its attachments which does not relate to MindNet's official business is neither given nor endorsed by us. We do not necessarily accept legal responsibility for the contents of this email or attachments. Views and opinions expressed are those of the author and do not necessarily represent those of MindNet.

**Comment by tanlee1** [comment]
I use EdgeGPT 0.1.18 solve this problem, thank you very much
> From:"Antonio ***@***.***>
> Date:Fri, Mar 24, 2023, 18:42
> Subject:Re: [acheong08/EdgeGPT] how solve this problem (Discussion #199)
> ***@***.***>
> ***@***.***>***@***.***>
> pip install git+https://github.com/acheong08/EdgeGPT.git
> is your cache isn't updating—
> Reply to this email directly,view it on GitHub<https://github.com/acheong08/EdgeGPT/discussions/199#discussioncomment-5416500>, orunsubscribe<https://github.com/notifications/unsubscribe-auth/A5HIWGDUD5XN7SW3R2VYTQLW5V3CBANCNFSM6AAAAAAWGH2EME>.
> You are receiving this because you authored the thread.[image: https://github.com/notifications/beacon/A5HIWGGXD5O7FUN2QQPHCA3W5V3CBA5CNFSM6AAAAAAWGH2EMGWGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3TUL5UWJTQAKKTDI.gif]Message ***@***.***>
Please consider the environment before printing this email
Mind Network (MindNet) can be contacted on ***@***.******@***.***>***@***.***>. This email and any attachments are confidential and may be protected by legal privilege. Any use of the information contained herein in any way (including, but not limited to, total or partial disclosure, reproduction, or dissemination) by persons other than the intended recipient(s) is prohibited. If you are not the intended recipient, you must contact us or the sender immediately and then securely delete the email and its attachments. We reserve all rights in respect of any unauthorised use of the email. Anything in this email or its attachments which does not relate to MindNet's official business is neither given nor endorsed by us. We do not necessarily accept legal responsibility for the contents of this email or attachments. Views and opinions expressed are those of the author and do not necessarily represent those of MindNet.
